### ISSUE TITLE
1.9.13: auoreconf fails because `AC_SYS_YEAR2038` acloacal macro is missing

### ISSUE THREAD

**Issue by kloczek** [sudo-project/sudo#242]
I cannot initialize source tree of the 1.9.13: because autoreconf cnnot find `AC_SYS_YEAR2038` acloacal macro.
```console
+ cd sudo-1.9.13
+ autoreconf -fiv
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'scripts'.
libtoolize: copying file 'scripts/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: aclocal --force
autoreconf: running: /usr/bin/autoconf --force
configure.ac:2442: error: possibly undefined macro: AC_SYS_YEAR2038
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: error: /usr/bin/autoconf failed with exit status: 1
```
Q: where is that macro? 🤔

**Comment by kanavin** [comment]
The macro has been added to autoconf recently, and there is not yet a tagged release that has it:
https://git.savannah.gnu.org/cgit/autoconf.git/commit/lib/autoconf/specific.m4?id=f6657256a37da44c987c04bf9cd75575dfca3b60

Certainly, not in the autoconf version that sudo requires:
```
AC_PREREQ([2.70])
```

I agree that relying on it in a released sudo version is premature, and should be reverted.

**Comment by kloczek** [comment]
I'm unig 2.71.
If it was not released I think that t would be good to add that macro to acinclude.m4.

**Comment by kanavin** [comment]
I'm using 2.71 too, the latest released version. The macro is not there.

**Comment by millert** [comment]
Sorry, I had expected autoconf 2.7.2 to be released already.  I just committed a fix to only call AC_SYS_YEAR2038 if it is defined.  Otherwise, it will use the method used in sudo 1.9.12.

**Comment by millert** [comment]
I plan to release 1.9.13p1 on Monday with this and other minor build-related fixes.

**Comment by kanavin** [comment]
Thanks!

**Comment by millert** [comment]
This is fixed in sudo 1.9.13p1, available now.

**Comment by kloczek** [comment]
Just tested p1 and looks like indeed issue is resolved 👍 
Thank you 😄
